torch/csrc/jit/operator_upgraders/version_map.cpp

```diff
@@ -96,5 +120,15 @@ void test_only_reset_flag() {
   isVersionMapSorted = false;
 }
 
+static bool calculatePackageVersionBasedOnUpgraders = false;
```

Comment:
Could you remind me what this is for?

Reply:
The temp fix we did was to still calculate version number for the package from the legacy map Mike introduced so that the overloads for div and full, that were not in our upgrader map, would be upgraded to correct version number. Since we figured it was risky to roll out a change involving version number, this flag is used to safely switch between the temp hack and the more correct approach where we add all the overloads in the version map. The end user can toggle this flag in the case of emergency.

Reply:
Ah, I remember now, just got thrown off by the name of "package"